### PR TITLE
Image setting updates

### LINKED ISSUE #2133: [Media sizing Phase 2] Quick wins: Slideshow, image banner, image with text
**Image sizing phase 2: Quick wins** 
- [ ] **Slideshow:** Change default slide height setting from "Adapt to first image" to "medium" [56 rem - no change to medium value]
- [ ] **Image banner:** Remove "Adapt section height to first image size" toggle and add as the first option under Banner height setting. (no change on default)
- [ ] **Image with text:** Add "medium" option to the image height setting. (no change on default) **Medium height values:** Desktop height/medium: 46rem minimum, Mobile height/medium: 29.6rem.

### PR DESCRIPTION
**Pull request by kmeleta**
### PR Summary: 

Makes minor adjustment to image size settings for slideshow, image banner, and image with text.

Important: Removed a setting in image banner.

### Why are these changes introduced?

Fixes #2133 

### What approach did you take?

- **Slideshow**: changed default value for `Slide height` setting from `Adapt` to `Medium`
- **Image banner**: Replaced `Adapt section height to first image size` checkbox setting with a new `Adapt to first image` option for existing `Banner height` select setting
- **Image with text**: Added new `Medium` option to existing `Image height` select setting

### Visual impact on existing themes

Slideshow: Default slide height has changed. This could cause a visual change for sections that don't have a value for this setting defined.

Image banner: The setting changes in image banner should be considered a breaking change. The same visual result is still achievable with the new option added to the banner height setting, but sections will need to be manually updated to use the new setting.

### Testing steps/scenarios
- Ensure setting options and defaults match description above and in the linked issue
- Verify new height setting for image with text works as expected
- Verify no unexpected regressions in updated sections
- Confirm all visual/breaking changes are accounted for in PR description

### Demo links

- [Editor](https://os2-demo.myshopify.com/admin/themes/137436397590/editor)

### Checklist
- [ ] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [ ] Requested review from UX (Only for changes that are affecting the experience or perceivable visual details)
- [ ] Created a ticket for the [help.shopify.com](https://help.shopify.com) documentation team about updates to theme settings. (Internal-only task)
- [ ] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [ ] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [ ] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [ ] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [ ] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

**Review by duygukalaycioglu — DISMISSED** [review]
Looking good! 👍

**Review by eugenekasimov — APPROVED** [review]
I think you could use the Prettier plugin even though I haven't merged it yet. 

The rest looks great!

File: sections/image-banner.liquid [hunk 1]
```diff
@@ -143,29 +143,26 @@
...
       "default": "medium",
       "label": "t:sections.image-banner.settings.image_height.label",
       "info": "t:sections.image-banner.settings.image_height.info"
     },
-    {
-      "type": "checkbox",
-      "id": "adapt_height_first_image",
-      "default": false,
-      "label": "t:sections.image-banner.settings.adapt_height_first_image.label",
-      "info": "t:sections.image-banner.settings.adapt_height_first_image.info"
```
Comment (LucasLacerdaUX) [inline sections/image-banner.liquid:167]: Should we remove these translation keys in `en.default.schema.json`?

**Review by LucasLacerdaUX — APPROVED** [review]
Works as expected! Just left two minor comments before you request translations 👍

File: sections/image-banner.liquid (hunk 1 above)
Comment (kmeleta) [reply sections/image-banner.liquid:196]: I actually did mean to initially. Do you know what happens if we do remove some? Do the generated strings end up getting removed from the other files async or anything or would we have to go and specifically delete them ourselves?

**Review by eugenekasimov — APPROVED** [review]

File: sections/image-banner.liquid (hunk 1 above)
Comment (LucasLacerdaUX) [reply sections/image-banner.liquid:196]: I'm pretty sure it gets removed by the translation platform at some point if we remove it from the English file. But I'm okay with doing it on a separate PR.

Comment (melissaperreault) [reply sections/image-banner.liquid:196]: Hey team! Seems like the translation was not removed. TIL!
There is an open PR for removing it from our community https://github.com/Shopify/dawn/pull/2482

**Pull request merged**